src/geovista/geodesic.py

```diff
@@ -279,20 +279,20 @@ def mesh(self) -> pv.PolyData:
 
         Examples
         --------
-        Add a bounding-box to the plotter for the ``africa`` panel of a cubed-sphere.
-        The geodesic bounding-box is generated from the 4 corners of the cubed-sphere
-        panel located over Africa. Natural Earth coastlines are also rendered along
+        Add a ``C32`` bounding-box around North Africa and Western Europe.
+        The geodesic bounding-box is generated by defining 4 longitude-latitude
+        corners. Natural Earth coastlines are also rendered along
         with a texture mapped Natural Earth base layer.
 
         >>> import geovista
-        >>> from geovista.geodesic import panel
+        >>> from geovista.geodesic import BBox
         >>> plotter = geovista.GeoPlotter()
         >>> _ = plotter.add_base_layer(
         ...     texture=geovista.natural_earth_hypsometric(), style="wireframe"
         ... )
-        >>> bbox = panel("africa", c=32)
+        >>> bbox = BBox(lons=[-20, 20, 20, -20], lats=[10, 10, 50, 50], c=32)
```

Comment:
The built-in panels are a neat feature, but I felt it was important to draw more attention to the flexibility of `BBox`. The built-in panels are still demonstrated in the example for `panel()`